addons/link_tracker/controller/main.py

```diff
@@ -21,3 +25,19 @@ def full_url_redirect(self, code, **post):
         if not redirect_url:
             raise NotFound()
         return request.redirect(redirect_url, code=301, local=False)
+
+    @http.route("/link_tracker/qrcode/<int:tracker_id>", auth="user")
```

Comment:
Can we use the generic route like other modules called "report/barcode/QR" ?

I let you check examples in the code, would be great to simplify.

Reply:
I think we can modify the base route and use the available "kwargs" to support a "download" parameter.

0 or 1, with default being 0 (same as "humanreadable").

If specified and equal to 1, it adds the header:
`("Content-Disposition", "attachment; filename=qrcode.png")`

I don't think we need to support a filename, seems a bit overkill, having the ID in the filename doesn't help the user much.
If you download multiple qrcode, your system will add "(1)", "(2)" etc, which achieves kind of the same purpose as the ID.

Seems like it would be a very light change and simplify your code a lot.

What do you think?